### ISSUE TITLE
Cannot find entry file src\renderer\index.ts

### ISSUE THREAD

**Issue by iMegaMaximus2** [electron-userland/electron-webpack#293]
```
Error: Cannot find entry file src\renderer\index.ts (or main.ts, or app.ts, or index.js, or main.js, or app.js)
```

How i can use custom index file? like `index.coffee` or `ololo.coffee` ?

**Comment by Nifled** [comment]
You can read about that in [this section in the docs](https://webpack.electron.build/configuration). You can specify your custom entry files through the `sourceDirectory` key.

**Comment by loopmode** [comment]
Have a look at this merged pull request: https://github.com/electron-userland/electron-webpack/pull/261
You can now specify a `renderer.template` option (released with 2.7.0)

- we still need to add documentation for this
- I am not sure whether..a typescript template will work.. @iMegaMaximus2 maybe you can try and report back?

**Comment by loopmode** [comment]
No activity, issue should be solved with https://github.com/electron-userland/electron-webpack/pull/261
Closing
